jsonschema-module-jackson/src/main/java/com/github/victools/jsonschema/module/jackson/JsonEnumDefaultValueResolver.java

```diff
@@ -0,0 +1,20 @@
+package com.github.victools.jsonschema.module.jackson;
+
+import com.fasterxml.jackson.annotation.JsonEnumDefaultValue;
+import com.github.victools.jsonschema.generator.TypeScope;
+import java.lang.reflect.Field;
+import java.util.Arrays;
+
+public class JsonEnumDefaultValueResolver {
+
+    public static String apply(TypeScope typeScope) {
+        if (typeScope.getType().getErasedType().isEnum()) {
+            return Arrays.stream(typeScope.getType().getErasedType().getDeclaredFields())
```

Comment:
**suggestion**: Since `MethodScope.getType()` can return `null` for `void` methods and you're only applying this to fields, I'd suggest making the declared parameter type `FieldScope` instead.
Also `getErasedType()` may return `null`, I believe, if the given type is merely a generic type placeholder.
```suggestion
    public static String apply(FieldScope scope) {
        Class<?> erasedType = scope.getType().getErasedType();
        if (erasedType != null && erasedType.isEnum()) {
            return Arrays.stream(erasedType.getDeclaredFields())
```